default-resources.yml

```diff
@@ -0,0 +1,178 @@
+resources:
+  Resources:
+    VPC:
+      Type: AWS::EC2::VPC
+      Properties:
+        CidrBlock: 172.98.0.0/18
+        EnableDnsSupport: true
+        EnableDnsHostnames: true
+        Tags:
+        - Key: Name
+          Value: fuzzbucket-${opt:stage, 'dev'}-vpc
+
+    PublicSubnet:
+      Type: AWS::EC2::Subnet
+      Properties:
+        VpcId: !Ref VPC
+        AvailabilityZone: ${aws:region, 'us-east-1'}a
+        CidrBlock: 172.98.0.0/19
+        MapPublicIpOnLaunch: true
+        Tags:
+        - Key: Name
+          Value: fuzzbucket-${opt:stage, 'dev'}-public
+
+    PublicRouteTable:
+      Type: AWS::EC2::RouteTable
+      Properties:
+        VpcId: !Ref VPC
+        Tags:
+        - Key: Name
+          Value: fuzzbucket-${opt:stage, 'dev'}-public
+
+    PublicRoute:
+      Type: AWS::EC2::Route
+      DependsOn:
+        - GatewayToInternet
+      Properties:
+        RouteTableId: !Ref PublicRouteTable
+        DestinationCidrBlock: 0.0.0.0/0
+        GatewayId: !Ref InternetGateway
+
+    PublicSubnetRouteTableAssociation:
+      Type: AWS::EC2::SubnetRouteTableAssociation
+      Properties:
+        SubnetId: !Ref PublicSubnet
+        RouteTableId: !Ref PublicRouteTable
+
+    InternetGateway:
+      Type: AWS::EC2::InternetGateway
+      Properties:
+        Tags:
+        - Key: Name
+          Value: fuzzbucket-${opt:stage, 'dev'}-ig
+
+    GatewayToInternet:
+      Type: AWS::EC2::VPCGatewayAttachment
+      Properties:
+        VpcId: !Ref VPC
+        InternetGatewayId: !Ref InternetGateway
+
+    S3Endpoint:
+      Type: AWS::EC2::VPCEndpoint
+      Properties:
+        RouteTableIds:
+        - !Ref PublicRouteTable
+        ServiceName:
+          Fn::Join:
+          - '.'
+          - - com.amazonaws
+            - Ref: AWS::Region
+            - s3
+        VpcId: !Ref VPC
+
+    FuzzbucketDefaultSecurityGroup:
+      Type: AWS::EC2::SecurityGroup
+      Properties:
+        GroupDescription: fuzzbucket default access
+        GroupName: fuzzbucket-${opt:stage, 'dev'}-default-sg
+        VpcId: !Ref VPC
+        SecurityGroupEgress:
+        - CidrIp: 0.0.0.0/0
+          Description: allow all outgoing
+          FromPort: 0
+          IpProtocol: -1
+          ToPort: 65535
+
+        SecurityGroupIngress:
+        - CidrIp: 0.0.0.0/0
+          Description: allow ssh incoming
+          FromPort: 22
```

Comment:
TODO: separate this into its own [resource](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-securitygroupingress.html) and guard it on a condition so that the security group is not flagged by Security Hub when configuring fuzzbucket in its most secure form

Reply:
My hope was that by breaking out the resources into a separate file that customization at the level of a different "resources" file would be acceptable. WDYT about locking down the `default-resources.yml` security groups to be secure by default, but with entries like these in comments with explanation?

Reply:
I am good with secure by default with example of how to customize